pipelines/deepvariant-based-germline-variant-calling-fq2vcf/Dockerfile_part1

```diff
@@ -28,7 +30,10 @@ RUN echo "source activate dv_env" >> ~/.bashrc
 RUN source ~/.bashrc
 
 WORKDIR /
-RUN git clone --recursive https://github.com/IntelLabs/Open-Omics-Acceleration-Framework.git
+#RUN git clone --recursive https://github.com/IntelLabs/Open-Omics-Acceleration-Framework.git
```

Comment:
delete this line

Reply:
In addition, I see some commented code in the Docker file part 2 and the Docker file of fq2sortedbam as well. Please delete those.

Reply:
Done, part2 has some original comments, kept them.